src/xdocs/config_misc.xml

```diff
@@ -69,6 +88,28 @@
   &lt;property name=&quot;javaStyle&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example:
+        </p>
+        <source>
+public class MyClass {
+  int[] nums; // violation
+  String strings[]; // OK
+
+  char[] toCharArray() { // OK
+    return null;
+  }
+
+  byte getData()[] { // violation
+    return null;
+  }
+}
+        </source>
+        <p>
+          Note that method return types must be Java style even when the check is configured
+          to enforce C style. This is because function/method return types in C/C++ cannot be
+          arrays, so there is no "C style" for such array types.
```

Comment:
This sounds like this should be in description and not down here in examples. See other note.